src/constants/constants.ts

```diff
@@ -0,0 +1,22 @@
+export const movieSections = [
+  {
+    id: '1',
+    sortBy: 'popularity.desc',
+    sectionTitle: 'Popular movies',
+  },
+  {
+    id: '2',
+    sortBy: 'vote_count.desc',
+    sectionTitle: 'Most viewed',
+  },
+  {
+    id: '3',
+    sortBy: 'primary_release_date.asc',
+    sectionTitle: 'Old is Gold',
+  },
+];
+
+export const ButtonModes = {
```

Comment:
constants should always be in capital cases so that we can differentiate with other constants/variables defined in other files.
rename to BUTTON_MODES